Dockerfile-linux.template

```diff
@@ -82,6 +83,17 @@ RUN set -eux; \
 	find /opt/pypy/lib-python -depth -type d -a \( -name test -o -name tests \) -exec rm -rf '{}' +; \
 	rm pypy.tar.bz2; \
 	\
+{{ if is_3 then ( -}}
```

Comment:
Do you think we should also gate on `7.3.3` so that it auto goes away for the next release or would you rather have a failing `update.sh` job?

Reply:
I'd prefer a failing `update.sh` job so we have a clear reminder/forcing function to actually come and remove it instead of carrying the cruft. :sweat_smile: